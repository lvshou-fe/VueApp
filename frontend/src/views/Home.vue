<template>
  <div class="home">
    <div class="homeinner">
      Homepage test
    </div>
  </div>
</template>

<script>
export default {
  name: 'home'
};
</script>

<style lang="scss" scoped>
.home {
  display: flex;
  flex-direction: column;
  flex: 1 1 auto;
}

.homeinner {
  display: flex;
  flex-direction: column;
  flex: 1 1 auto;
  align-items: center;
  justify-content: center;
  padding: 0 30px 30px 30px;
}
</style>
