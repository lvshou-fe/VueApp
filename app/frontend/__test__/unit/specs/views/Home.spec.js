import { shallow } from '@vue/test-utils';
import Home from '@/views/Home';

describe('Home.vue', () => {
  it('should render', () => {
    const wrapper = shallow(Home);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
