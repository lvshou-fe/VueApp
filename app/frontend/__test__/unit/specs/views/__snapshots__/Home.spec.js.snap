// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home.vue should render 1`] = `
<div class="home">
  <div class="homeinner">
    Homepage test
  </div>
</div>
`;
